Instruments/microwave_tuning_gui.py

```diff
@@ -213,9 +216,12 @@ def regen_plots(self):
                 # }}}
             self.axes.plot(xx/1e6, yy, 'r', alpha=0.5)
             self.axes.set_xlabel(r'$\nu_{B12}$ / GHz')
-            self.axes.set_ylabel(r'Rx / mV')
+            self.axes.set_ylabel(r'Rx / dBm')
             self.axes.axvline(x=dip_frq_GHz, ls='--', c='r')
-            self.B12.set_freq(dip_frq_GHz*1e9) # always do this, so that it should be safe to slightly turn up the power
+            self.myconfig['uw_dip_center_GHz'] = self.dip_frq_GHz
+            self.myconfig['carrierfreq_mhz'] = self.dip_frq_GHz * self.myconfig['guessed_mhz_to_ghz']
+            self.myconfig.write()
+            self.B12.set_freq(self.dip_frq_GHz*1e9) # always do this, so that it should be safe to slightly turn up the power
```

Comment:
edits the uw_dip_center_GHz parameter in ini file and adjusts the carrier frequency based off of the current guessed_MHz_to_GHz parameter (ppt)